src/rss-blogs-en.njk

```diff
@@ -30,6 +30,7 @@
   {%- set absolutePostUrl = post.url | absoluteUrl(metadata.url) %}
   <entry>
     <title>{{ post.data.title }}</title>
+    <author>{{ post.data.author }}</author>
```

Comment:
Does this have the email address or `email address (name)`?

See: https://www.rssboard.org/rss-specification#ltauthorgtSubelementOfLtitemgt

Reply:
This has the human readable name that the author entered in the `author:` field of the blog, so in the case of Rosita's last blog "Rosita". Do you think we should leave it out?

Reply:
I think the spec wants it to be an email or email (name), so yeah, I think we should leave it out.

See: https://www.xn--8ws00zhy3a.com/blog/2007/04/rss-email-addresses
See: https://trac.edgewall.org/ticket/3836
See: https://invisioncommunity.com/forums/topic/447623-include-author-name-in-rss-feeds/